include/LightGBM/feature_group.h

```diff
@@ -181,6 +185,8 @@ class FeatureGroup {
   std::unique_ptr<Bin> bin_data_;
   /*! \brief True if this feature is sparse */
   bool is_sparse_;
+  /*! \brief Threshold for treating a feature as a sparse feature */
+  double sparse_threshold_;
```

Comment:
@huanzhang12 Sorry, I didn't see it.
Ok,  I think we can put it into ```Dataset``` class to avoid the duplicate.